lib/common/xml.c

```diff
@@ -1001,74 +1001,6 @@ crm_xml_set_id(xmlNode *xml, const char *format, ...)
     free(id);
 }
 
-/*!
- * \internal
- * \brief Get consecutive bytes encoding non-ASCII UTF-8 characters
- *
- * \param[in] text  String to check
- *
- * \return Number of non-ASCII UTF-8 bytes at the beginning of \p text
- */
-static size_t
-utf8_bytes(const char *text)
```

Comment:
If you want to be paranoid, require CHAR_BIT to be 8 in configure.ac

Reply:
good idea